src/charm.py

```diff
@@ -295,13 +296,6 @@ def _on_install(self, event) -> None:
             auth=auth, machine_ip=self._unit_ip(self.unit), config=self.mongodb_config
         )
 
```

Comment:
I think we should add pbm user before calling `self.client_relations.oversee_users` in `_initialise_replica_set` to make pbm user created during the init.